ui/src/api/googleOauth.ts

```diff
@@ -28,31 +22,28 @@ export const openGoogleOAuthPage = () => {
 
 /**
  * Exchange google's access token for our API's access token.
- * @param googleAccessToken - google's access token
+ * @param authorizationCode - Authorization code returned by google
  * @param redirectToHome - callback function to redirect to home
  */
 export const convertAccessToken = async (
-  googleAccessToken: string,
+  authorizationCode: string,
   redirectToHome: () => void,
   failRedirect: () => void
 ) => {
+  console.log(authorizationCode);
```

Comment:
Remove console.log

Reply:
This should probably be caught in linting test. Can you figure out if there's a way?

Reply:
Yeah, we can add this in `.eslintrc.js`.
```javascript
...
  rules: {
    'no-console': 2,
...
```

Reply:
Create an issue for this.